<ui version="4.0" stdsetdef="1" >
  <author>J Adamczewski/S Linev</author>
  <comment>Event information </comment>
  <exportmacro></exportmacro>
  <class>TGo4EventInfo</class>
  <widget class="QGo4Widget" name="TGo4EventInfo" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>372</width>
        <height>150</height>
      </rect>
    </property>
    <property name="caption" >
      <string>Event Information</string>
    </property>
    <property name="icon" >
      <pixmap>:/icons/go4logo2_big.png</pixmap>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>7</number>
      </property>
      <property name="spacing" >
        <number>3</number>
      </property>
      <property name="resizeMode" >
        <enum>Minimum</enum>
      </property>
      <item row="1" column="0" >
        <widget class="Q3GroupBox" name="MbsPrintBox" >
          <property name="title" >
            <string>MBS Event</string>
          </property>
          <layout class="QHBoxLayout" >
            <item>
              <widget class="QPushButton" name="PrintEventButton" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize" >
                  <size>
                    <width>30</width>
                    <height>30</height>
                  </size>
                </property>
                <property name="maximumSize" >
                  <size>
                    <width>30</width>
                    <height>30</height>
                  </size>
                </property>
                <property name="text" >
                  <string/>
                </property>
                <property name="icon" >
                  <iconset>:/icons/analysiswin.png</iconset>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>Print MBS event on analysis terminal (analysis should run)</string>
                </property>
              </widget>
            </item>
            <item>
              <spacer name="spacer17_2_2_2_2_4_2" >
                <property name="sizeHint" >
                  <size>
                    <width>16</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                </property>
              </spacer>
            </item>
            <item>
              <widget class="QSpinBox" name="PrintEventNum" >
                <property name="maximum" >
                  <number>9999</number>
                </property>
                <property name="singleStep" >
                  <number>10</number>
                </property>
                <property name="value" >
                  <number>1</number>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>number of events to be printed out on analysis terminal</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QSpinBox" name="PrintEventSid" >
                <property name="specialValueText" >
                  <string>all</string>
                </property>
                <property name="maximum" >
                  <number>99999999</number>
                </property>
                <property name="minimum" >
                  <number>-1</number>
                </property>
                <property name="value" >
                  <number>-1</number>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>Choose mbs subevent id for printoug</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="LongBox" >
                <property name="text" >
                  <string>long</string>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>Check this to print mbs event contents in longwords</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="HexBox" >
                <property name="text" >
                  <string>hex</string>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>Check this to print mbs event contents in hex format</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item row="2" column="0" >
        <layout class="QHBoxLayout" >
          <item>
            <widget class="QPushButton" name="RefreshButton" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="minimumSize" >
                <size>
                  <width>30</width>
                  <height>30</height>
                </size>
              </property>
              <property name="maximumSize" >
                <size>
                  <width>30</width>
                  <height>30</height>
                </size>
              </property>
              <property name="text" >
                <string/>
              </property>
              <property name="icon" >
                <iconset>:/icons/zoom.png</iconset>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Inspect Sample of current event</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer7" >
              <property name="sizeHint" >
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QCheckBox" name="RemoteButton" >
              <property name="text" >
                <string>ShowRemote</string>
              </property>
              <property name="checked" >
                <bool>true</bool>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Check this to have printout of event sample in analysis terminal.</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer6" >
              <property name="sizeHint" >
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QCheckBox" name="TreeButton" >
              <property name="text" >
                <string>TTreeSample</string>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Check this to use temporary tree for scanning event structure</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item row="0" column="0" >
        <layout class="QHBoxLayout" >
          <item>
            <widget class="QLabel" name="Label1" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>1</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text" >
                <string>Event:</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLabel" name="EventLbl" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text" >
                <string>EventLbl</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer16" >
              <property name="sizeHint" >
                <size>
                  <width>222</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="MbsButton" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="minimumSize" >
                <size>
                  <width>30</width>
                  <height>30</height>
                </size>
              </property>
              <property name="maximumSize" >
                <size>
                  <width>30</width>
                  <height>30</height>
                </size>
              </property>
              <property name="text" >
                <string/>
              </property>
              <property name="icon" >
                <iconset>:/icons/mbslogo.png</iconset>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Work with MBS event</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="local" >QGo4Widget.h</include>
  </includes>
  <connections>
    <connection>
      <sender>RefreshButton</sender>
      <signal>clicked()</signal>
      <receiver>TGo4EventInfo</receiver>
      <slot>RefreshClicked()</slot>
    </connection>
    <connection>
      <sender>PrintEventButton</sender>
      <signal>clicked()</signal>
      <receiver>TGo4EventInfo</receiver>
      <slot>PrintEventClicked()</slot>
    </connection>
    <connection>
      <sender>MbsButton</sender>
      <signal>clicked()</signal>
      <receiver>TGo4EventInfo</receiver>
      <slot>MbsButton_clicked()</slot>
    </connection>
  </connections>
</ui>
